modules/apps/web-experience/journal/journal-web/src/main/resources/META-INF/resources/view_entries.jsp

```diff
@@ -27,7 +27,7 @@ String searchContainerId = ParamUtil.getString(request, "searchContainerId");
 %>
 
 <liferay-ui:search-container
-	emptyResultsMessage="no-web-content-was-foundd"
+	emptyResultsMessage="No web content was found"
```

Comment:
Hard code error message is not correct way to fix this bug. It will not work in multi languages.